src/classes/CRLP_RecalculateBTN_CTRL.cls

```diff
@@ -138,18 +138,25 @@ public with sharing class CRLP_RecalculateBTN_CTRL {
             if (this.objType == Contact.SObjectType) {
                 calculateContactSoftCredit(recordId);
                 RLLP_OppRollup.rollupContactsFuture(new Set<Id>{ recordId });
+
             } else if (this.objType == Account.SObjectType) {
                 RLLP_OppRollup.rollupAccountsFuture(new Set<Id>{ recordId });
 
             } else if (this.objType == General_Accounting_Unit__c.SObjectType) {
                 ALLO_Rollup_SCHED rollup = new ALLO_Rollup_SCHED(recordId);
                 rollup.runRollups();
-            } else if (this.objType == npe03__Recurring_Donation__c.SObjectType) {
+
+            } else if (this.objType == npe03__Recurring_Donation__c.SObjectType
```

Comment:
When enhanced RDs are enabled and a user disables Customizable Rollups (even though it is not recommended), then legacy rollups are not calculated for the enhanced RDs.
@force2b , @rponti-sforg

Reply:
Makes sense. The `RD_RecurringDonations.updateRecurringDonationOnOppChange()` method does a similar check and would just return if CRLP is enabled. I wonder if that method should also check for and return if RD2 is enabled.

Reply:
The change has been made directly in the method to check for RD2 enablement.